modules/snowflake_connector_config/snowflake-connector-config.tpl

```diff
@@ -0,0 +1,19 @@
+{
+  "buffer.count.records": "1000",
+  "buffer.flush.time": "60",
+  "buffer.size.bytes": "5000000",
+  "connector.class": "com.snowflake.kafka.connector.SnowflakeSinkConnector",
+  "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+  "name": "${connector_name}",
+  "snowflake.database.name": "${snowflake_database}",
+  "snowflake.private.key": "${snowflake_private_key}",
+  "snowflake.private.key.passphrase": "${snowflake_private_key_passphrase}",
+  "snowflake.schema.name": "PUBLIC",
```

Comment:
With this setup, we would have a database per data provider/producer.

Is that the intention? I actually don't know the best layout, and we can use this for now (it should be easy to change).